utilities/ttl/ttl_test.cc

```diff
@@ -40,7 +40,7 @@ class SpecialTimeEnv : public EnvWrapper {
   int64_t current_time_ = 0;
 };
 
-class TtlTest : public testing::Test {
+class TtlTest : public testing::TestWithParam<bool> {
```

Comment:
Having every test to run twice just covering the way of closing it feels an overkill to me.
Can we only do this in one of the tests?